.gitignore

```diff
@@ -34,6 +34,8 @@ deepfence_frontend/product_version.txt
 
 deepfence_agent/tools/apache/scope/proto/
 deepfence_agent/tools/apache/scope/vendor/
+deepfence_agent/fargate/build
```

Comment:
nit: I think we should stop using fargate as a name internally, it should be more generic, like host or standalone or binaries